pkg/license/catalog.go

```diff
@@ -31,16 +31,20 @@ import (
 // CatalogOptions are the spdx settings
 type CatalogOptions struct {
 	CacheDir string // Directrory to catch the license we download from SPDX.org
+	Version  string // Version of the licenses to download  (eg v3.19) or blank for latest
 }
 
 // DefaultCatalogOpts are the predetermined settings. License and cache directories
 // are in the temporary OS directory and are created if the do not exist
-var DefaultCatalogOpts = &CatalogOptions{}
+var DefaultCatalogOpts = &CatalogOptions{
+	Version: "v3.20",
```

Comment:
Where this comes from? Worth add a comment?

Reply:
Sure thing. Ultimately [we want to embed the compressed SPDX license list in the binary](https://github.com/kubernetes-sigs/bom/issues/44) so we would default to the version included, but still let users specify another one.

I've added more context in the code to explain a little bit more.